content/en/tracing/other_telemetry/connect_logs_and_traces/opentelemetry.md

```diff
@@ -43,14 +41,13 @@ from opentelemetry import trace
 class CustomDatadogLogProcessor(object):
     def __call__(self, logger, method_name, event_dict):
         # An example of adding datadog formatted trace context to logs
-        # from: https://github.com/open-telemetry/opentelemetry-python-contrib/blob/b53b9a012f76c4fc883c3c245fddc29142706d0d/exporter/opentelemetry-exporter-datadog/src/opentelemetry/exporter/datadog/propagator.py#L122-L129 
+        # from: https://github.com/open-telemetry/opentelemetry-python-contrib/blob/b53b9a012f76c4fc883c3c245fddc29142706d0d/exporter/opentelemetry-exporter-datadog/src/opentelemetry/exporter/datadog/propagator.py#L127-L129 
         current_span = trace.get_current_span()
         if not current_span.is_recording():
             return event_dict
 
         context = current_span.get_span_context() if current_span is not None else None
         if context is not None:
-            event_dict["dd.trace_id"] = str(context.trace_id & 0xFFFFFFFFFFFFFFFF)
             event_dict["dd.span_id"] = str(context.span_id)
```

Comment:
```suggestion
            event_dict["dd.trace_id"] = str(context.trace_id)
            event_dict["dd.span_id"] = str(context.span_id)
```

Reply:
This code snippet only shows an example of setting trace id and span ids. We can also include remappings for `service`, env`, and `version` (using the OTEL Resource SDK)

Reply:
@mabdinur , can you provide that code please?